haskell/platforms/list.bzl

```diff
@@ -0,0 +1,44 @@
+OS = {
+    "aix": None,
+    "darwin": "@bazel_tools//platforms:osx",
+    "dragonfly": None,
+    "freebsd": "@bazel_tools//platforms:freebsd",
+    "haiku": None,
+    "hpux": None,
+    "ios": "@bazel_tools//platforms:ios",
+    "linux_android": "@bazel_tools//platforms:android",
+    "linux": "@bazel_tools//platforms:linux",
+    "mingw32": "@bazel_tools//platforms:windows",
```

Comment:
I don't know ghc naming convention, but I'm surprised to find `mingw32` in `OS` instead of `mingw` and 32bits later in `ARCH`. If that't correct, please add a one line comment.